### PR TITLE
Added new icons

### LINKED ISSUE #161: Add Dribbble icon
It would be a nice addition to having the Dribbble icon within this package.

### PR DESCRIPTION
**Pull request by ShimiSun**
New icons:
* Dripple
* Ebay
* Figma
* LastFm

Tested with storybook, looks good.

Fixes #161 #160 #158 #162

### CODE REVIEW

File: src/js/__tests__/__snapshots__/Components-test.js.snap [hunk 1]
```diff
@@ -174,10 +174,12 @@ Object {
   "Download": [Function],
   "Drag": [Function],
   "Drawer": [Function],
+  "Dribbble": [Function],
```
Comment (ericsoderberghp) [inline src/js/__tests__/__snapshots__/Components-test.js.snap]:
```suggestion
  "Dribble": [Function],
```

File: public/img/dribbble.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```
Comment (ericsoderberghp) [inline public/img/dribbble.svg]: dribbble.svg -> dribble.svg

**Comment by LRNZ09** [comment]
Hey @ShimiSun @ericsoderberghp I'm almost sure that Dribbble (with triple b) is the correct name, see https://dribbble.com/

**Comment by ShimiSun** [comment]
@LRNZ09 lol, thanks for letting us know, we'll fix it

**Pull request merged**